### PR TITLE
Update CLI in refactor namespace to work in deployed env

### PR DESCRIPTION
**Pull request by mjgiarlo**
# Why was this change made?

I have tested these changes by running the get, update, and register operations in QA.

# How was this change tested?

CI

### CODE REVIEW

File: lib/sdr_client/redesigned_client/cli.rb [hunk 1]
```diff
@@ -64,17 +64,26 @@ def update(druid)
         validate_druid!(druid)
         # Make sure client is configured
         client
-        job_id = CLI::Update.run(druid, **options)
+        job_id = Update.run(druid, **options)
```
Comment (mjgiarlo) [inline lib/sdr_client/redesigned_client/cli.rb:67]: No need to have the `CLI` module prefix here.

File: lib/sdr_client/redesigned_client/cli.rb [hunk 2]
```diff
@@ -64,17 +64,26 @@ def update(druid)
...
           return
         end
 
+        # the extra args to `say` prevent appending a newline
+        say('SDR is processing your request.', nil, false)
+
         job_status = client.job_status(job_id: job_id)
-        if job_status.wait_until_complete
-          say "success! (druid: #{job_status.druid})"
+        job_status.wait_until_complete { say('.', nil, false) }
```
Comment (mjgiarlo) [inline lib/sdr_client/redesigned_client/cli.rb:77]: Make a new CLI look like the old one.

File: lib/sdr_client/redesigned_client/cli.rb [hunk 3]
```diff
@@ -64,17 +64,26 @@ def update(druid)
...
+        if job_status.complete?
+          if job_status.errors
+            say_error " errored! #{job_status.errors}"
+          else
+            say " success! (druid: #{job_status.druid})"
+          end
         else
-          say_error "errored! #{job_status.errors}"
+          say_error " job #{job_id} did not complete\n#{job_status.result.inspect}"
         end
```
Comment (mjgiarlo) [inline lib/sdr_client/redesigned_client/cli.rb:87]: Make the new CLI look like the old one.

File: lib/sdr_client/redesigned_client/job_status.rb [hunk 4]
```diff
@@ -35,11 +37,26 @@ def errors
...
                               timeout_in_secs: 180,
                               backoff_factor: 2.0,
                               max_secs_between_requests: 60)
-        poll_until_complete(secs_between_requests, timeout_in_secs, backoff_factor, max_secs_between_requests)
+        begin
+          Timeout.timeout(timeout_in_secs) do
+            loop do
+              break if complete?
+
+              yield if block_given?
```
Comment (mjgiarlo) [inline lib/sdr_client/redesigned_client/job_status.rb:49]: The only consumer that sends a block is the CLI, which prints dots to show the user that the polling is in progress.

File: lib/sdr_client/redesigned_client/metadata.rb [hunk 5]
```diff
@@ -35,6 +35,7 @@ def deposit # rubocop:disable Metrics/MethodLength
           model: request_builder.to_cocina,
           basepath: basepath,
           files: files,
+          accession: accession,
```
Comment (mjgiarlo) [inline lib/sdr_client/redesigned_client/metadata.rb:38]: Bug discovered by testing the CLI in QA.

**Review by justinlittman — APPROVED** [review]

**Pull request merged**